src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -44,14 +50,27 @@ public void processPullRequest(JsonNode payload) {
     }
 
     private static FormatConfig readConfig() {
+        String url = Utils.readTemplateConfigUrlFromProperties(System
+                .getProperty(Utils.JBOSS_CONFIG_DIR), Utils.CONFIG_FILE);
+
         String configFileName = System.getProperty(TEMPLATE_FORMAT_FILE);
         if (configFileName == null) {
             configFileName = System.getProperty(Utils.JBOSS_CONFIG_DIR) + "/format.yaml";
         }
         log.info(configFileName);
         File configFile = new File(configFileName);
-        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
 
+        if (url != null && !configFile.exists()) {
+            try (InputStream inputStream = new URL(url).openStream();
+                 OutputStream outputStream = new FileOutputStream(configFile)) {
+                byte[] buffer = new byte[inputStream.available()];
```

Comment:
this is wrong, check the second paragraph in the method javadoc... https://docs.oracle.com/javase/7/docs/api/java/io/InputStream.html#available()